src/daily-auths-report.tsx

```diff
@@ -145,18 +145,32 @@ function DailyAuthsReport(): VNode {
     ref?.current?.appendChild(
       Plot.plot({
         y: {
-          tickFormat: format(".2s"),
+          tickFormat: format(".1s"),
         },
         style: {},
         marks: [
           Plot.ruleY([0]),
-          Plot.barY(data || [], {
-            x: "date",
-            y: "count",
-            fill: agency ? "friendly_name" : "agency",
-            title: (d: ProcessedResult) => (agency ? d.friendly_name : d.agency),
-            filter: (d: ProcessedResult) => d.ial === ial && (!agency || d.agency === agency),
-          }),
+          Plot.rectY(
+            data || [],
+            Plot.binX(
+              {
+                y: "sum",
+              },
+              {
+                y: "count",
+                x: {
+                  value: "date",
+                  thresholds: utcDay,
```

Comment:
Checking my understanding: According to [the documentation](https://www.npmjs.com/package/@observablehq/plot), this is passed...

>three arguments: the array of input values, the domain minimum, and the domain maximum.

Is that what we're expecting here?

Aside: I notice for shorter timespans, it can add a mid-day label. Are we okay with that?

![image](https://user-images.githubusercontent.com/1779930/131682543-d8aa1734-c9ca-4553-b155-6741e12da288.png)

Reply:
> Checking my understanding: According to [the documentation](https://www.npmjs.com/package/@observablehq/plot), this is passed...
> 
> > three arguments: the array of input values, the domain minimum, and the domain maximum.

<img width="789" alt="Screen Shot 2021-09-01 at 8 36 10 AM" src="https://user-images.githubusercontent.com/458784/131701082-dd951cc0-edb2-425b-a546-336fd5b5fcea.png">

I guess the specific type isn't clearly, but I think that "a time interval (for temporal binning)" means an interval function like from d3-time



> Aside: I notice for shorter timespans, it can add a mid-day label. Are we okay with that?

Normally it would be just fine to let D3 auto-pick the interval, however, for shorter spans, it picks 12-hour interval and unforunately our data is only at midnight so we get these weird gaps:

<img width="663" alt="Screen Shot 2021-09-01 at 8 40 15 AM" src="https://user-images.githubusercontent.com/458784/131701685-a837980c-ff0a-4c00-9b3b-f267429d6923.png">

so I worked around that by pinning to `utcDay` specifically to avoid that. I think the labels might keep some of that same previous logic, and I think it's okay for now

Reply:
>I guess the specific type isn't clearly, but I think that "a time interval (for temporal binning)" means an interval function like from d3-time

Ah, yeah, that was a bit unclear. Although, checking again, I think I assumed `utcDay` was a function, when in fact it's an object with `count` and `every`. Just a misread on my part, makes sense now 👍